tavern/test_data.go

```diff
@@ -4,24 +4,24 @@ import (
 	"context"
 
 	"github.com/kcarretto/realm/tavern/ent"
-	"github.com/kcarretto/realm/tavern/ent/credential"
+	// "github.com/kcarretto/realm/tavern/ent/credential"
```

Comment:
I think we should delete this instead of commenting just to avoid dead code piling up.

Reply:
But I love dead code :(